tests/test_models.py

```diff
@@ -453,3 +453,10 @@ def test_population_config_order_equal(self):
                 config.metadata["name"] for config in model.population_configurations]
             for p, c in zip(pop_names, config_names):
                 self.assertEqual(p, c)
+
+    # Test that we are indeed getting a valid DDB back
+    # admittedly a pretty bad test...
+    def test_demography_debugger(self):
+        for model in stdpopsim.all_demographic_models():
+            ddb = model.get_demography_debugger()
+            self.assertEqual(type(ddb), msprime.DemographyDebugger)
```

Comment:
You could instead use `self.assertIsInstance()`.

Reply:
sure no prob